puppet/spec/classes/jenkins_node_spec.rb

```diff
@@ -31,7 +31,7 @@
           it { is_expected.to contain_file('/etc/pbuilder/bullseye64/hooks/F66-add-nodesource-nodistro-repos').with_ensure('absent') }
           it { is_expected.to contain_file('/etc/pbuilder/jammy64/hooks/F66-add-nodesource-repos').with_ensure('absent') }
           it { is_expected.to contain_file('/etc/pbuilder/jammy64/hooks/F66-add-nodesource-nodistro-repos').with_ensure('present') }
-          it { is_expected.to contain_file('/etc/pbuilder/bookworm64/hooks/F66-add-nodesource-repos').with_ensure('absent') }
+          it { is_expected.to contain_file('/etc/pbuilder/bookworm64/hooks/F66-add-nodesource-repos').with_ensure('present') }
           it { is_expected.to contain_file('/etc/pbuilder/bookworm64/hooks/F66-add-nodesource-nodistro-repos').with_ensure('absent') }
```

Comment:
I would have expected the "nodistro" one to be present now.
Looking at https://github.com/theforeman/foreman-infra/blob/1485c8265e29408081a9d69272550d7cec0cd7de/puppet/modules/jenkins_node/manifests/pbuilder_setup.pp#L44-L45 we need to adjust the "release" part there too :/
(the other repo is not updated anymore and we shouldn't be using it at all for current releases -- we should clean that up)

Reply:
the Debian cleanup: https://github.com/theforeman/foreman-infra/pull/2275